### PR TITLE
Add wallet and clean up UI

### PR DESCRIPTION
**Pull request by kev-ngo**
## Changes
- Minor styling changes
- Added wallet

## TODO
- Bring wallet's address into the useEffect hooks that bring in token balance

## Screenshots
unknown

### CODE REVIEW

File: src/Home.tsx [hunk 1]
```diff
@@ -38,14 +42,21 @@ export const Home: React.FC = () => {
...
   const w3Client = useWeb3ApiClient();
+  const { active, activate, account, deactivate } = useWeb3React();
 
   const logMsgHandler = async (): Promise<any> => {
     setSubmitted(true);
   };
 
+  // Need help setting account from connected wallet
+  const setConnectHandler = async (): Promise<any> => {
+    setAccountAddress(account);
```
Comment (Too-Far) [inline src/Home.tsx:56]: You can try and use this ```
const [accountNum] = useState<string>(
        account ? `${account.substring(0, 6)}...${account.substring(account.length - 4)}` : '',
    );
``` It lets the user see the last four of their address

**Review by Too-Far — COMMENTED** [review]
Looks good to me, other than the one comment. IS this supposed to automatically trigger a query when the users account is loaded? When I tested locally id did not.

File: src/Home.tsx (hunk 1 above)
Comment (Niraj-Kamdar) [reply src/Home.tsx:53]: You also need to use `setSubmitted(true)` so that user don't need to submit manually after connecting the wallet.

**Comment by Too-Far** [comment]
@kev-ngo  and @Niraj-Kamdar I fixed the conflicts and made it so the user's token info automatically gets fetched once they authorize the wallet.

**Review by Niraj-Kamdar — APPROVED** [review]
LGTM!

**Pull request merged**